augly/video/__init__.py

```diff
@@ -47,11 +47,12 @@
     vflip,
     vstack,
 )
-
 from augly.video.transforms import (
+    FPS,
```

Comment:
Why did you move this one? It should go back in alphabetical order as it was before.

Reply:
Ah I pushed my code because I thought I was going to get a bug fix so I didn't format it with `black` before